bog_app/app/models/creature.rb

```diff
@@ -0,0 +1,8 @@
+class Creature < ActiveRecord::Base
+  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
+  validates_attachment_content_type :avatar, content_type: /\Aimage\/.*\Z/
+  def self.search(search)
+    where("name ILIKE ?", "%#{search}%")
+    where("content ILIKE ?", "%#{search}%")
```

Comment:
Your creatures don't have an attribute called content. Perhaps this was supposed to be description? I think that this is the second issue that's preventing your search from working.